### PR TITLE
Feature/header mobile

### PR DESCRIPTION
**Pull request by JFozzie**
**Header mobile version**

Add the mobile version with the hamburger menu, it appears on the left and blocks the scroll of the content below and only the modal continues with scroll

**Branch:**
feature/headerMobile

**Test page:**
localhost:8080/headermenu.html

**Example:**
image

### CODE REVIEW

File: src/scss/components/_header.scss [hunk 1]
```diff
@@ -10,40 +10,205 @@
...
+    flex-direction: column;
+    gap: 0;
+    justify-content: space-between;
+    left: -100%;
+    padding: 4.8rem 2.4rem;
+    position: fixed;
+    top: 70px;
+    transition: 0.3s;
+    width: 100%;
+    height: 90vh;
```
Comment (jrmahechad) [inline src/scss/components/_header.scss:37]: I think this should be 100vh.

File: src/scss/components/_header.scss [hunk 2]
```diff
@@ -10,40 +10,205 @@
...
   }
 
   &__menu {
-    align-items: center;
+    background-color: var(--white);
     display: flex;
-    gap: 4rem;
+    flex-direction: column;
+    gap: 0;
+    justify-content: space-between;
```
Comment (jrmahechad) [inline src/scss/components/_header.scss:30]: I think you can remove this. The space between items is defined by the margins.

File: templates/header.hbs [hunk 3]
```diff
@@ -1,19 +1,59 @@
...
+            <i class="donor__socials__icons fa fa-instagram"></i>
+          </a>
+          <a href="http://www.youtube.com">
+            <i class="donor__socials__icons fa fa fa-youtube-play"></i>
+          </a>
+        </div>
+      </div>
+    </ul>
+
+    <div class="hamburger">
```
Comment (jrmahechad) [inline templates/header.hbs:51]: Can you change this `div` for a `button`? Also move it before `header__menu__donor-info`?

File: templates/header.hbs [hunk 4]
```diff
@@ -1,19 +1,59 @@
...
+    <ul class="header__menu">
       {{#each mainLinks}}
-        <li class="header__menu__item" >
+        <li class="header__menu__item">
           {{{module this}}}
         </li>
       {{/each}}
-      </ul>
+      <div class="header__menu__donor-info">
+        <h5>Donar para mas inclusión</h5>
```
Comment (jrmahechad) [inline templates/header.hbs:24]: @jsebax Do we have this and all info below in the CMS?

Comment (jrmahechad) [reply templates/header.hbs:24]: @JFozzie can you add classes to elements inside header__menu__donor-info?

**Review by jrmahechad — CHANGES_REQUESTED** [review]
@JFozzie great work!!

File: src/scss/components/_header.scss (hunk 2 above)
Comment (JFozzie) [reply src/scss/components/_header.scss:30]: I tried removing the gag too and there is no problem, thanks

File: templates/header.hbs (hunk 4 above)
Comment (JFozzie) [reply templates/header.hbs:24]: The donor component is not done yet, this info is temporal.

File: templates/header.hbs (hunk 3 above)
Comment (JFozzie) [reply templates/header.hbs:51]: The change from div to button, i got it, and  it's done.
but move this before `header__menu__donor-info` I don't understand!

**Review by jrmahechad — APPROVED** [review]

**Review by jcmonsa — APPROVED** [review]

**Pull request merged**